molecule/cookiecutter/scenario/verifier/goss/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/verify.yml

```
@@ -10,37 +10,58 @@
   vars:
     goss_version: v0.3.6
     goss_arch: amd64
-    goss_dst: /usr/local/bin/goss
-    goss_sha256sum: 53dd1156ab66f2c4275fd847372e6329d895cfb2f0bcbec5f86c1c4df7236dde
-    goss_url: "https://github.com/aelsabbahy/goss/releases/download/{{ goss_version }}/goss-linux-{{ goss_arch }}"
-    goss_test_directory: /tmp
+    goss_bin: /usr/local/bin/goss
+    goss_sha256sum: 53dd1156ab66f2c4275fd847372e6329d895cfb2f0bcbec5f86c1c4df7236dde.
```

Is that trailing dot a typo?
```suggestion
    goss_sha256sum: 53dd1156ab66f2c4275fd847372e6329d895cfb2f0bcbec5f86c1c4df7236dde
```

---

Hi, thanks, I removed the dot. Although I think it was there when I updated that sha256sum. However, it's gone now :)